config/locales/en.yml

```diff
@@ -301,6 +301,7 @@ en:
     resources:
       destroy:
         notice: File has been deleted
+        error: File can't be deleted. There is article which uses it.
```

Comment:
`File cannot be deleted. There is an article which uses it.`

Reply:
Updated.